src/BuildScriptGenerator/Helpers/ListBlobsHelper.cs

```diff
@@ -2,22 +2,41 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 // --------------------------------------------------------------------------------------------
+using System;
 using System.Linq;
+using System.Net.Http;
 using System.Xml.Linq;
 using Microsoft.Oryx.BuildScriptGenerator.Common;
 
 namespace Microsoft.Oryx.BuildScriptGenerator
 {
     internal static class ListBlobsHelper
     {
-        public static XDocument GetAllBlobs(string sdkStorageBaseUrl, string platform, System.Net.Http.HttpClient httpClient, string oryxSdkStorageAccountAccessToken)
+        public static XDocument GetAllBlobs(string sdkStorageBaseUrl, string platform, HttpClient httpClient, string oryxSdkStorageAccountAccessToken)
         {
             var oryxSdkStorageAccountAccessArgs = oryxSdkStorageAccountAccessToken?.TrimStart(new char[] { '?' }) ?? string.Empty;
 
             var url = string.Format(SdkStorageConstants.ContainerMetadataUrlFormat, sdkStorageBaseUrl, platform, string.Empty, oryxSdkStorageAccountAccessArgs);
-            var blobList = httpClient
-                .GetStringAsync(url)
-                .Result;
+            string blobList;
+            try
+            {
+                blobList = httpClient
+                    .GetStringAsync(url)
+                    .Result;
+            }
+
+            // AggregateException is thrown by .Result
+            catch (AggregateException ae)
+            {
+                throw new AggregateException(
+                        $"Http request to '{sdkStorageBaseUrl}' failed. {Constants.NetworkConfigurationHelpText}\n{ae}");
```

Comment:
Please consider using Environment.NewLine rather than hardcoding "\n". Especially if in the future we switch between Windows and Linux for some stuff (I know it wouldn't impact here, but you see what I mean), having a consistent way to do things will make sure we don't have surprises on one OS and not in the other.